content/11-serverless/04-traditional-vs-serverless-databases.mdx

```diff
@@ -0,0 +1,126 @@
+
+High scalability also means that you can experiment with applications easily and move from testing to production without changes to your database service.  You can set up testing and staging databases that will scale down to zero when not in use.  This can be especially useful when practicing continuous integration and continuous delivery because your pipelines can use a database service that perfectly mirrors your production environment while only paying for the small amount of usage your tests require.
+
+#### Reduced management responsibility
+
+Perhaps the biggest highlight for serverless databases is the way they can help offload management responsibilities from your team.  While many organizations have gotten use to the level of administrative overhead required by managed databases, serverless databases take on additional levels of support and management to further reduce your workload.
+
+Serverless database services not only manage the infrastructure and database management system software, they also manage resource allocation and many of the policies for your databases.  Instead of having to figure out exactly how much storage your database requires or how many database instances are required to serve your average traffic levels, the system can just respond to the conditions it finds.  Instead of managing scaling rules, you can instead just set boundaries for the levels of scaling you desire and the platform will intelligently scale within those constraints.
+
+This sounds similar to how managed databases are often described, but in practice it ends up being significantly different.  You don't have to make many decisions ahead of time with serverless databases.  Mainly, you configure any thresholds for costs that you want to maintain and any specific tweaks to the scaling you want to make, like always having at least one query executor available, for instance.  The system handles nearly everything else to ensure that your application's database usage is always available, performing up to standards, and responding to the current request environment.
+
+## Conclusion
+
+In this article, we took a look at some of the key differences between traditional, self-managed databases and serverless database services.  We talked about how they are different from a user's perspective and what trade-offs you must consider when deciding on a database strategy depending on your organization's priorities.
+
+While serverless databases are not suitable for every type of application, they can make database management and operation significantly easier for many organizations.  Understanding when the might be a good solution and what shortcomings they might have can help you figure out if they can and should be considered when evaluating database solutions.
```

Comment:
Might be good to outlink to the prisma data platform at the end for serverless users or those considering serverless.

Reply:
Great idea. Thanks for the reminder on that one.